packages/esm-patient-common-lib/src/orders/types.ts

```diff
@@ -15,6 +15,7 @@ export interface OrderBasketItem {
       };
     };
   };
+  extractedOrderError?: { message: string; fieldErrors: string[]; globalErrors: string[] };
```

Comment:
This type should probably also be extracted into a named type, since its the return type of `extractErrorDetails`